src/stdune/hashtbl.ml

```diff
@@ -83,6 +83,12 @@ module Make(H : sig
     | Some p -> Result.Error p
 
   let keys t = foldi t ~init:[] ~f:(fun key _ acc -> key :: acc)
+
+  let to_dyn f t =
+    Dyn.Map (
+      foldi t ~init:[] ~f:(fun key data acc ->
```

Comment:
Might be worth sorting by the keys here.

Reply:
Since this is just a hash table, I'm afraid there's no good suitable comparison function. Do you mean to use polymorphic comparison or by the hash perhaps?

Reply:
Ah, that's unfortunate. 
I guess we could sort by the Dyn of the keys. 
I think that's better than hash comparison and than polymorphic comparison.
Feel free to do it or not.

Reply:
I added it. It's just a one liner, not a huge deal.